quad_x0: 0.0
quad_y0: 0.0
quad_z0: 3.0

goal_x: 3.0
goal_y: 3.0
goal_z: 3.0

link1: 0.175
link2: 0.42

te: 10
dt: 0.1

